### ISSUE TITLE
Add a section at the end on extending the suzanne example

### ISSUE THREAD

**Issue by ottworks** [rust-tutorials/learn-luminance#15]
I've completed the tutorial, but I don't know where to go from there. How do I render two monkeys? How can I give each monkey its own independent transform? How should I structure an entity system around this api? The AST examples in luminance-rs's readme sound like what I want to do, but I don't know how to translate them into code.

**Comment by phaazon** [comment]
Thanks a lot for the feedback. I have several potential other tutorials, but be sure I’ll add some more very quick! I’ll come back to you with a more complete answer ASAP when I have more spare-time. :)

For rendering several monkeys with independent transform, you can have a look at `Buffer` and the concept of tessellation instancing. I’ll try to write the next chapter on that! :)

**Comment by ottworks** [comment]
Thanks! I'm interested in multiple separate models as well though, and I'm not really sure if I should just append their triangles or do a whole separate pipeline

**Comment by phaazon** [comment]
Yeah, there are several possible options there. I’ll try to describe them as accurately as possible! :)

**Comment by phaazon** [comment]
Closing as I’m going to rewrite everything.
